build/index.ts

```diff
@@ -9,6 +9,7 @@ clean('./dist')
       ['./src/collection.json', './dist'],
       ['./src/**/schema.json', './dist'],
       ['./src/prettier/templates/.prettierrc', './dist'],
```

Comment:
This line was accidentally committed before.
Can you remove this line?

Reply:
I tried, but the glob ignores dot-files

Reply:
Ah, found out the dot-files are ignored by default. But they can be included: https://github.com/mrmlnc/fast-glob#dot

Will try that.

Reply:
I already configured globby to accept dot-Files
Thanks for the hint!

@Strulle It was no bug, just a feature. 😆